mip/gurobi.py

```diff
@@ -535,13 +535,10 @@ def get_objective(self) -> LinExpr:
         st = GRBgetdblattrarray(self._model, attr, 0, self.num_cols(), obj)
         if st != 0:
             raise ParameterNotAvailable("Error getting objective function")
-        obj_expr = xsum(
-            obj[i] * self.model.vars[i]
-            for i in range(self.num_cols())
-            if abs(obj[i]) > 1e-20
-        )
-        obj_expr.const = self.get_objective_const()
```

Comment:
would you please check what happens just changing this line
```
obj_expr.add_const(self.get_objective_const())
```

Reply:
Sure, that works too but we should probably also fix the line below that. Not sure whether it's the preferrable way, but I can push it. Also, another bug: CBC's `get_objective` method does not (in contrast to Gurobi's) set `objective.sense`.

Reply:
We had a similar discussion before, where the const setting was missing anyhow at gurobi. We decided not to create a new linear expression.

Then please open a new issue if you have found another bug. We are trying to fix the objective things here and then we see next.